tests/errors/cpp_object_template.pyx

```diff
@@ -18,7 +18,7 @@ def memview():
     vmv.push_back(array.array("i", [1,2,3]))
 
 _ERRORS = u"""
-10:16: Python object type 'Python object' cannot be used as a template argument
-12:16: Python object type 'A' cannot be used as a template argument
-17:16: Reference-counted type 'int[:]' cannot be used as a template argument
+10:15: Python object type 'Python object' cannot be used as a template argument
+12:15: Python object type 'A' cannot be used as a template argument
+17:15: Reference-counted type 'int[:]' cannot be used as a template argument
```

Comment:
Looking at the code, the column should be 17, not 15.

Reply:
The error was coming from the `TemplatedTypeNode` (i.e. pointing at the bracket, with 0-based indices). I'll change to to point to the template argument, which I think is an improvement (although not strictly related to this PR)